pkg/kv/kvserver/store_raft.go

```diff
@@ -305,6 +310,9 @@ func (s *Store) HandleRaftUncoalescedRequest(
 		// that dropping the request is safe. Raft will retry.
 		s.metrics.RaftRcvdDropped.Inc(1)
 		s.metrics.RaftRcvdDroppedBytes.Inc(size)
+		if logRaftRecvQueueFullEvery.ShouldLog() {
+			log.Warningf(ctx, "raft receive queue for r%d is full", req.RangeID)
```

Comment:
The log throttling here is global, so if multiple ranges are struggling it's essentially random who's going to get logged. Should we throttle on a per-range/replica basis?